Bricks/file/test.cc

```diff
@@ -293,30 +292,105 @@ TEST(File, ScanDirParameters) {
   FileSystem::MkDir(d, FileSystem::MkDirParameters::Silent);
 
   {
-    std::set<std::string> xs;
-    FileSystem::ScanDir(base_dir, [&xs](const std::string& x) { xs.insert(x); });
-    EXPECT_EQ("f", current::strings::Join(xs, ','));
+    std::set<std::string> names;
+    std::map<std::string, const FileSystem::ScanDirItemInfo> item_infos;
```

Comment:
There's no profit in using `const` value type in containers here.
Removing it will make the code below more concise and readable.